community/kernel/src/main/java/org/neo4j/procedure/Procedure.java

```diff
@@ -101,4 +101,15 @@
 @Retention( RetentionPolicy.RUNTIME )
 public @interface Procedure
 {
+    /**
+     * The namespace and name for the procedure, as a period-separated
```

Comment:
Hmm, so you could have `Namespace` annotated on the class, and `Name` on the procedure method, for instance?